### DISCUSSION TITLE
Question regarding "reading borehole survey data" (unknown model in real case studies)

### DISCUSSION THREAD

**Discussion by WJX97** [gempy-project/gempy#948] [Q&A] [answered]
Hello,

An error occurred while running the code for "Create the Survey object":    ValueError: all inc values must be in range 0 <= inc < 180.
Upon checking, I found that all the inc values are -0.000001.

I tried manually correcting the negative values in the inc column, or deep copying the data survey_df, but neither approach resolved the issue.
I'm guessing that the problem might occur during the data processing inside the Survey.from_df(). 

Any guidance or possible solutions would be greatly appreciated. Thank you!

![1](https://github.com/user-attachments/assets/7cf623aa-eafb-4b51-be91-0b6a6891b91c)
![1729786617566](https://github.com/user-attachments/assets/ef7b9004-4523-429c-ae01-7d498d2bfa84)
![1729786785709](https://github.com/user-attachments/assets/5ab71eb7-b417-4be8-bf37-adb57122241e)

**Comment by javoha** [comment, marked as answer]
Hi,

this seems to be an external example and the error occirs in a subsurface function that is unrelated to gempy as far as I can see,
I think it would be best to post your question on their git: https://github.com/softwareunderground/subsurface

Cheers,
Jan

**Reply by WJX97** [reply]
Hello Jan,

Thank you very much for the clarification.

Kind regards,
Jingxiao
